_posts/2019-02-26-bazel-0.23.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: posts
+title: "Bazel 0.23"
+authors:
+  - laurentlb
+---
+
+[Bazel 0.23.0](https://github.com/bazelbuild/bazel/releases/tag/0.23.0) has
+arrived with an exciting mix of new features, important bug fixes, and updated
+documentation.
+
+This version of Bazel has been regression-tested against 51 open-source
+projects, including Gerrit, Protocol Buffers, TensorFlow and the full suite of
+rules hosted in bazelbuild's GitHub organization. This makes it our most
+comprehensively tested release yet. Want to get your open-source project on
+Bazel's CI to ensure that we catch any regressions or have suggestions what we
+should test against? Reach out to
+[bazel-dev@googlegroups.com](mailto:bazel-discuss@googlegroups.com) and we can
```

Comment:
looks like there's already a template in bazelbuild/continuous-integration:

https://github.com/bazelbuild/continuous-integration/issues/new/choose